webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml

```diff
@@ -31,9 +31,14 @@
         template="system:/groovy/portal/webui/container/UIContainer.gtmpl"
         cssClass="border-box-sizing">
         <access-permissions>Everyone</access-permissions>
-        <container id="topNewsPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-          <access-permissions>*:/platform/users</access-permissions>
-        </container>
+        <portlet-application>
+          <portlet>
+            <application-ref>social-portlet</application-ref>
+            <portlet-ref>ImagesPortlet</portlet-ref>
+          </portlet>
+          <access-permissions>Everyone</access-permissions>
+          <show-info-bar>false</show-info-bar>
+        </portlet-application>
         <container id="newsBodyPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
           <access-permissions>Everyone</access-permissions>
           <container cssClass="newsBodyPageLeftColumn"
```

Comment:
Change newsBodyPageRightColumn and newsBodyPageLeftColumn from UIContainer.gtmpl to UISimpleColumnContainer.gtmpl

In that case I think that added [css classes](https://github.com/exoplatform/news/commit/af3952fe2d05f25d7c2dde134340c67ecd32a3cd#diff-77f3cfb6a6caf540496eab73a1012e948b54b7b600c2510c14e9298f93a13ba7) are no more needed